test/e2e/storage/testsuites/driveroperations.go

```diff
@@ -107,7 +105,7 @@ func GetSnapshotClass(
 			"metadata": map[string]interface{}{
 				// Name must be unique, so let's base it on namespace name and use GenerateName
 				// TODO(#96234): Remove unnecessary suffix.
-				"name": names.SimpleNameGenerator.GenerateName(ns + "-" + suffix),
+				"name": names.SimpleNameGenerator.GenerateName(ns),
```

Comment:
> // TODO(#96234): Remove unnecessary suffix.

Remove the comment as well ?

Reply:
> > https://github.com/kubernetes/kubernetes/pull/97183/files#diff-f18ed9fa9279e79977b207010ee6601f97178ed4a5720610bd67275ef5a50ce3R107
> > // TODO(#96234): Remove unnecessary suffix.
> 
> Remove the comment as well ?

Ok, I have removed it and updated the code. @pacoxu